reaper/src/util.ts

```diff
@@ -0,0 +1,69 @@
+import { S3 } from '@aws-sdk/client-s3';
+import { SSM } from '@aws-sdk/client-ssm';
+import { fromIni, fromNodeProviderChain } from '@aws-sdk/credential-providers';
+import {
+	AWS_REGION,
+	buildRecordsBucketName,
+	ENV_VAR_BATCH_SIZE,
+	REAPER_APP_NAME,
+	STACK,
+} from '../../cdk/bin/constants';
+
+export const getBatchSize = () => {
+	const value = process.env[ENV_VAR_BATCH_SIZE as string];
+	if (!value) {
+		throw Error(
+			`Missing environment variable '${ENV_VAR_BATCH_SIZE as string}'`,
+		);
+	}
+	return value;
+};
+
+const LOCAL_PROFILE = 'media-service';
+
+const IS_RUNNING_LOCALLY = !process.env.LAMBDA_TASK_ROOT;
+
+const stage = process.env.STAGE ?? 'TEST';
+
+const standardAwsConfig = {
+	region: AWS_REGION as string,
+	credentials: IS_RUNNING_LOCALLY
+		? fromIni({ profile: LOCAL_PROFILE })
+		: fromNodeProviderChain(),
+};
+
+const ssm = new SSM(standardAwsConfig);
+
+const paramStorePromiseGetter =
+	(WithDecryption: boolean) => (nameSuffix: string) => {
+		const Name = `/${stage}/${STACK as string}/${
```

Comment:
```suggestion
		const name = `/${stage}/${STACK as string}/${
```

Reply:
this naming is so it can be used shorthand for the input to the AWS SSM operation (which annoyingly takes params with capitalised first letter)